### PR TITLE
Xdr malware fixes

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue - 1 ](https://jira-hq.paloaltonetworks.local/browse/CIAC-3408)
          [link to the issue - 2](https://jira-hq.paloaltonetworks.local/browse/CIAC-3376)

## Description
Fixes XDR file retrieval logic bugs 

## Screenshots
N.A.
## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Endpoint_Investigation.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@ contentitemexportablefields:
   contentitemfields:
     propagationLabels:
     - all
-description: "This playbook handles all the endpoint investigation actions available with Cortex XSIAM, including the following tasks:\n
+description: " This playbook is part of the 'Malware Investigation And Response' flow. For more information, please refer to https://xsoar.pan.dev/docs/reference/packs/malware-investigation-and-response.
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Endpoint_Investigation.yml:5]: Please change to pack and not flow - in all relevant files and PR's

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Endpoint_Investigation_README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+This playbook is part of the 'Malware Investigation And Response' flow. For more information, please refer to https://xsoar.pan.dev/docs/reference/packs/malware-investigation-and-response.
 This playbook handles all the endpoint investigation actions available with Cortex XSIAM, including the following tasks:
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Endpoint_Investigation_README.md:2]: Need to change here the second sentence as well (the cortex XSIAM)

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Retrieve_File_by_sha256.yml [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+      brand: ''
+    nexttasks:
+      '#default#':
+      - '8'
+      yes:
+      - '4'
+    separatecontext: false
+    conditions:
+    - label: yes
+      condition:
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Retrieve_File_by_sha256.yml:324]: The conditional question and the path for "Yes" are inconsistent.

**Review by altmannyarden — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

**Pull request merged**